_data/nav.yml

```diff
@@ -17,6 +17,10 @@
     title: Meet the Bugzilla team.
     name: Meet the Team
 
+  - link: /about/zarroboogs.html
+    title: About Zarro Boogs Corporation
```

Comment:
Title is meant to add context, let's tell people what that Corporation is in short terms here

Reply:
"Find out about the company managing the Bugzilla Project."  That matches the style of the titles on other things in that menu.